ui/eslint.config.js

```diff
@@ -8,12 +8,7 @@ const config = defineConfig([
   ...createConfig(opts),
   {
     rules: {
-      'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/control-has-associated-label': 'off',
-      'react/destructuring-assignment': 'off',
```

Comment:
Lots of `this.props.<propName>` everywhere, so impractical to fix. These violations can usually be fixed when refactoring class components to function components.